packages/prettier-plugin-java/scripts/clone-samples.js

```diff
@@ -9,65 +9,65 @@ const samplesDir = path.resolve(__dirname, "../samples");
 const core = [
   {
     repoUrl: "https://github.com/jhipster/jhipster-sample-app",
-    branch: "v7.9.3"
+    branch: "main"
```

Comment:
I prefer to keep a fixed version, in order to have more reproducible builds/tests

Reply:
Agreed! I originally decided to go with the way things were prior to #588, in case there was a reason that was originally done that I wasn't aware of. Just pushed this change! 👍